Chat: update Edit and Delete Messages demo description + title (#30040)
DevExpress/DevExtreme: apps/demos/Demos/Chat/MessageEditing/description.md

> The Chat component allows to edit and delete messages sent by the current user.

Reviewer: ```suggestion
The Chat component allows you to edit and delete messages sent by the current user.
```

> Use a [data source](/Documentation/ApiReference/UI_Components/dxChat/Configuration/#dataSource) if you want users to edit and delete messages. Implement [custom CRUD operations](/Documentation/ApiReference/Data_Layer/CustomStore/). Once you configured these operations, enable [editing](/Documentation/ApiReference/UI_Components/dxChat/Configuration/editing/).

Reviewer: ```suggestion
Use a [data source](/Documentation/ApiReference/UI_Components/dxChat/Configuration/#dataSource) to allow users to edit and delete messages. Implement [custom CRUD operations](/Documentation/ApiReference/Data_Layer/CustomStore/). Once you configured these operations, enable [editing](/Documentation/ApiReference/UI_Components/dxChat/Configuration/editing/).
```
How does the second sentence in this paragraph relate to the first? Why do they need to implement custom CRUD operations? Be specific.

<!--split-->

[note] Chat does not update the data source automatically. Implement a [CustomStore](/Documentation/ApiReference/Data_Layer/CustomStore/) with CRUD operations to handle updates.

> The **editing** object includes [allowUpdating](/Documentation/ApiReference/UI_Components/dxChat/Configuration/editing/#allowUpdating) and [allowDeleting](/Documentation/ApiReference/UI_Components/dxChat/Configuration/editing/#allowDeleting) properties. These Boolean options are initially `false`. You can set them to `true` or use functions with custom logic. When options are `true`, users can edit/delete their own messages.

Reviewer: ```suggestion
The **editing** object includes [allowUpdating](/Documentation/ApiReference/UI_Components/dxChat/Configuration/editing/#allowUpdating) and [allowDeleting](/Documentation/ApiReference/UI_Components/dxChat/Configuration/editing/#allowDeleting) properties. These Boolean options are set to `false` initially. To allow your users to edit and delete their own messages, set these Boolean options to `true` or use functions with custom logic. 
```

> In this demo, try deleting or editing messages. First, check that the options are active in the panel next to the Chat component. Right-click (Control+Click on MacOS) or long-tap a message to open the context menu. Select "Delete" to remove the message, which then shows a marker in the feed. Choose "Edit" to view the original message and update its content. Click "Send" to save changes, and the message is marked as edited.

Reviewer: I do not understand what this paragraph is asking. Are you telling the reader to play around with this demo?

Author: Yes

Reviewer: Use this demo to review how to delete or edit messages. First, check that the options (what options? specify) are active in the panel next to the Chat component. Right-click (Control+Click on MacOS) or long-tap a message to open the context menu. Select "Delete" to remove the message, which (which what? specify) then shows a marker in the feed. Choose "Edit" to view the original message and update its content. Click "Send" to save changes; this will mark the message as edited.

Although I made a few edits above, there is still clarification that I need from you.

Reviewer: Use this demo to review how to delete or edit messages. First, ensure that "Options" are active in the panel next to the Chat component. Right-click (Control+Click on MacOS) or long-tap a message to open the context menu. Select "Delete" to remove the message; a marker is then shown in place of the deleted message in the feed. Choose "Edit" to view the original message and update its content. Click "Send" to save changes; this will mark the message as edited.
